doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-create-proc.adoc

```diff
@@ -11,9 +11,9 @@ With this data, the test scenario can validate the expected and actual results f
 
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project name.
-. Click *Add Asset* -> *Test Scenario*.
-. Enter an informative *Test Scenario* name and select the appropriate *Package*. The package that you specify must be the same package where the required rule assets have been assigned or will be assigned. You can import data objects from any package into the asset's editor.
-+
+. Click *Add Asset* -> *Test Scenario (Legacy)*.
+. Enter an informative *Test Scenario* name and select the appropriate *Package*. The package that you specify must be the same package where the required rule assets have been assigned or will be assigned. You can import data objects from any package into the asset's designer.
+. Select *RULE* as the *Source type*.
```

Comment:
This is not correct. There is no `Source type` option in the legacy test scenario

Reply:
Corrected as suggested.